feature-libs/order/styles/components/order-history/_index.scss

```diff
@@ -8,7 +9,6 @@
 @import './return-request-totals';
 @import './cancel-order';
 @import './consignment-tracking';
-@import 'cart';
```

Comment:
if this was to work, it was not a great solution as we would have been providing the entire cart styles from order.

Potential solutions:
- webpack to resolve for local develop in order to actually use @spartacus/cart, which will work in customer's app. But again, it causes issues bringing the entire cart styles until we properly split styles, which is in our backlog
- move the specific cart item styles in the @spartacus/styles, so that it can be available and re-used throughout features
- copy paste the styles in order lib and re-use it there.

Reply:
additional ideas:
- using classes instead placeholder selector, components from order lib already use cart lib, so we are ensured that it should be loaded already. Styles specified for components, probably will be in another class, then the element will have 2 classes: `cx-cart-item-list cx-specified-list`)
- keep placeholder selectors which would be nice to override in specified patch: `@spartacus/cart/shared`

Reply:
I'll drop fixing it in this PR as I saw it will be worked on your PR https://github.com/SAP/spartacus/pull/15564